diff --git a/node_modules/react-native-safe-area-context/ios/Fabric/RNCSafeAreaProviderComponentView.mm b/node_modules/react-native-safe-area-context/ios/Fabric/RNCSafeAreaProviderComponentView.mm
index f315ccd..95e911d 100644
--- a/node_modules/react-native-safe-area-context/ios/Fabric/RNCSafeAreaProviderComponentView.mm
+++ b/node_modules/react-native-safe-area-context/ios/Fabric/RNCSafeAreaProviderComponentView.mm
@@ -106,9 +106,9 @@ - (void)invalidateSafeAreaInsets
         .insets =
             {
                 .top = safeAreaInsets.top,
-                .left = safeAreaInsets.left,
-                .bottom = safeAreaInsets.bottom,
                 .right = safeAreaInsets.right,
+                .bottom = safeAreaInsets.bottom,
+                .left = safeAreaInsets.left,
             },
         .frame =
             {
